Demonstrate a routing issue with sanic 21.3.2, sanic-routing 0.5.2.

Tested with python 3.8.8.

`pipenv sync -d`

`pipenv run pytest`

Or run the server manually and use curl:

`pipenv run python -m app.run`

`./curl.sh`